pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -183,6 +177,16 @@ func NewController(
 		},
 	})
 
+	c.buildInformerFactory = &duck.CachedInformerFactory{
+		Delegate: &duck.EnqueueInformerFactory{
+			Delegate: &duck.TypedInformerFactory{},
```

Comment:
we'll want to pass `duck.TypedInformerFactory` in from `main.go`, but this otherwise looks right.